Addition of drink questions and drink continuation.

jayzamazing opened the pull request:
This covers the bartender asking what kind of drink the person wants along with asking the user if they want another drink and generating a random drink.

jayzamazing committed:
Added handlebars to bower.json. Changed json questions section to have question objects with keys, added list of drink adjectives and nouns to questionsandingredients.json. Added title, question-template and another-drink-template to index.html. Added settemplate, question-section on submit functions, added getquestionnumber and setquestionnumber to bartender, and getingredientlistnumber and setingredientlistnumer to pantry, and added check that callback is function for getquestionsingredients function in app.js. Added style.css file and styles for centering objects.

jayzamazing committed:
Added text align to newdrink and current drink in style.css. Added name of drink in another-drink-template in index.html. Added setstartover function, another-drink-form submit, getrandomnumber, resetquestionnumber to bartender prototype, getdrinkadjective/noun to pantry, and removed get/setingredientlistnumber from pantry in app.js.

aric87 commented on css/style.css:
```diff
@@ -0,0 +1,13 @@
+.answer-choices label {
+    margin-left: 0 !important;
```
Consider whether you _need_ !important here, or could you change the specificity. !important had its place, but often leads to a heap of them

aric87 commented on index.html:
```diff
@@ -1,14 +1,82 @@
+    <script src="bower_components/handlebars/handlebars.js"></script>
+    <script src="js/app.js"></script>
+    <script id="question-template" type="text/x-handlebars-template">
+        <form action="" id="question-form">
```
If you don't need an action, remove the attribute.

aric87 commented on index.html:
```diff
@@ -1,14 +1,82 @@
+                <h2>{{drink}}</h2>
+            </div>
+        </div>
+        <form action="" id="another-drink-form">
```
Ditto

aric87 commented on index.html:
```diff
@@ -1,14 +1,82 @@
+            <div class="row">
+                <div class="col-xs-12 col-sm-12 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
+                    <div class="col-centered new-drink-submit col-lg-2">
+                        <input type="submit" name="submit" class="btn btn-default btn-primary col-lg-12" value="YARRR!">
```
This needs a sound effect! :)

jayzamazing committed:
Added pirate-bar.jpg and yarr.wav. Added background image, transparency to jumbotron, and override label and button colors in style.css. Added audio to index.html. Added playmusic function, added check that one item is selected in app.js.

jayzamazing commented:
!important is being used to override bootstrap implementation. Added sound effect and background image, although I am not that great when it comes to visual stuff. Removed action.

jayzamazing committed:
Added description to readme.md.
